web/views/run_view.ex

```diff
@@ -1,3 +1,8 @@
 defmodule Gatherto.RunView do
   use Gatherto.Web, :view
+
+  def google_maps_api do
+    api_key = System.get_env("GOOGLE_MAPS_API_KEY")
```

Comment:
I don't know how I feel about this being the way it is in the RunView module. (Doesn't feel right)